captain-definition

```diff
@@ -0,0 +1,4 @@
+ {
+  "schemaVersion": 2,
+  "dockerfilePath": "./Dockerfile"
```

Comment:
I thought about it more and seems like it should use this

https://caprover.com/docs/captain-definition-file.html#use-image-name

to reference the pre-built images in which case just paste the four lines of definition file into the caprover UI.